### PR TITLE
Add: Monthshort, Dayshort to support locale Indonesian

### PR DESCRIPTION
**Pull request by muhammadridho**
Add Monthshort, Dayshort to support locale Indonesian.

### CODE REVIEW

File: src/locale/id.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ const locale = {
   name: 'id',
   weekdays: 'Minggu_Senin_Selasa_Rabu_Kamis_Jumat_Sabtu'.split('_'),
   months: 'Januari_Februari_Maret_April_Mei_Juni_Juli_Agustus_September_Oktober_November_Desember'.split('_'),
+  weekdaysShort: 'Min_Sen_Sel_Rab_Kam_Jum_Sab'.split('_'),
+  monthsShort: 'Jan_Feb_Mar_Apr_Mei_Jun_Jul_Agu_Sep_Okt_Nov_Des'.split('_'),
+  weekdaysMin: 'Mi_Sn_Sl_Rb_Km_Jm_Sb'.split('_'),
```
Comment (iamkun) [inline src/locale/id.js]: Mi -> Mg?

Comment (muhammadridho) [reply src/locale/id.js]: ah... thats good to hear

**Comment by iamkun** [comment]
Thanks.

**Comment by muhammadridho** [comment]
you're welcome

File: src/locale/id.js [hunk 2]
```diff
@@ -4,6 +4,9 @@ const locale = {
   name: 'id',
   weekdays: 'Minggu_Senin_Selasa_Rabu_Kamis_Jumat_Sabtu'.split('_'),
   months: 'Januari_Februari_Maret_April_Mei_Juni_Juli_Agustus_September_Oktober_November_Desember'.split('_'),
+  weekdaysShort: 'Min_Sen_Sel_Rab_Kam_Jum_Sab'.split('_'),
```
[the remark this answers is not in the document]
Comment (muhammadridho) [reply src/locale/id.js]: i just updated

**Pull request merged**